src/tools/jsondocck/src/main.rs

```diff
@@ -205,7 +205,21 @@ fn check_command(command: Command, cache: &mut Cache) -> Result<(), CkError> {
                     let val = cache.get_value(&command.args[0])?;
                     let results = select(&val, &command.args[1]).unwrap();
                     let pat = string_to_value(&command.args[2], cache);
-                    results.contains(&pat.as_ref())
+                    let has = results.contains(&pat.as_ref());
+                    // Give better error for when @has doesnt conatian
```

Comment:
```suggestion
                    // Give better error for when @has check fails
```
Although I kind of question whether the comment is necessary, I think this is reasonably self-explanatory.